dj/construction/exceptions.py

```diff
@@ -37,7 +37,7 @@ def set_raise(self, raise_: bool):
         """
         self._raise = raise_
 
-    def append(self, error: DJError, message: Optional[str]):
+    def append(self, error: DJError, message: Optional[str] = None):
```

Comment:
thx

Reply:
No problem but I'm pretty sure that's a bug I introduced when I replaced the context manager with this append method 😅